src/Inventory/Asset/Printer.php

```diff
@@ -359,36 +336,6 @@ public function handleMetrics()
         }
     }
 
-    /**
-     * Try to know if printer need to be updated from discovery
-     * Only if IP has changed
-     * @return boolean
-     */
-    public static function needToBeUpdatedFromDiscovery(CommonDBTM $item, $val)
```

Comment:
public methods should not be removed in bugfixes versions.

```php
/**
 * @FIXME Remove method in GLPI 10.1.
 */
public static function needToBeUpdatedFromDiscovery(CommonDBTM $item, $val)
{
    return false;
}
```

Reply:
done